iocBoot/iocMuonJaws/config.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
+<macros>
+<macro name="LVDCOM_OPTIONS" pattern="^\d?\d?$" description="Options that define how the VI is started. Add selected options together to make the final value. 1 - warn if idle, 2 - start if idle, 4 - stop VIs if started on exit, 8 - stop VI on exit. (Default 1)" />
```

Comment:
Should this not contain the regular things such as PORT, BAUD, BITS, PARITY etc.

Reply:
See comment above.